extensions/infinispan-client/deployment/src/main/java/io/quarkus/infinispan/client/deployment/InfinispanClientProcessor.java

```diff
@@ -591,7 +590,7 @@ static <T> SyntheticBeanBuildItem configureAndCreateSyntheticBean(String name,
     static <T> SyntheticBeanBuildItem configureAndCreateSyntheticBean(RemoteCacheBean remoteCacheBean, Supplier<T> supplier) {
         SyntheticBeanBuildItem.ExtendedBeanConfigurator configurator = SyntheticBeanBuildItem.configure(RemoteCache.class)
                 .types(remoteCacheBean.type)
-                .scope(Singleton.class) // Some Infinispan API won't work if this is not a Mock
+                .scope(ApplicationScoped.class)
```

Comment:
@gsmet this is a necessary breaking change for people using Query in an old fashion way. Documentation is provided in the extension ref doc, but I think we need to add a line about it in the 3.9 release notes